pkg/controller/controller_broker_test.go

```diff
@@ -96,6 +96,18 @@ func TestShouldReconcileBroker(t *testing.T) {
 			interval:  3 * time.Hour,
 			reconcile: false,
 		},
+		{
+			name: "ready, interval not elapsed, checksum changed",
+			broker: func() *v1alpha1.Broker {
+				broker := getTestBrokerWithStatus(v1alpha1.ConditionTrue)
+				cs := "22081-9471-471"
+				broker.Status.Checksum = &cs
+				return broker
+			}(),
+			now:       time.Now(),
+			interval:  3 * time.Hour,
+			reconcile: true,
```

Comment:
I was hoping the name here would suffice. The only thing it's doing differently from the existing test cases is setting a checksum.